### PR TITLE
Bounty tables & calculator updates

### PR DESCRIPTION
**Pull request by jessiwright**
Per request from Brian Reigersberg
3 pages updated
Reference document: https://docs.google.com/document/d/14yh-wGAdANRDcWvZyjmrTv6ZUqvPlt_NXYfXnGPOAHY/edit#

### CODE REVIEW

**Review by Reigertje — APPROVED** [review]

File: docs/programs/bounties.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ If you're unsure of how much to award the hacker, you can communicate a suggeste
 3. Select **Set award**.
 4. Select **Suggest amount** in the next action picker.
 
-![suggesting bounties](./images/bounties-suggest.png)
+![suggesting bounties](./images/bounties-2.png)
```
Comment (Reigertje) [inline docs/programs/bounties.md:40]: This image shouldn't change

File: docs/programs/bounties.md [hunk 2]
```diff
@@ -78,3 +78,9 @@ cURL | Ruby
 After the bounty has been awarded, the hacker will receive an email to claim the bounty. HackerOne will collect the person's tax form before processing the payout. The awarded amount, including your applicable fees, will be deducted from your balance immediately. A resolved dummy report will show up in your Bugs overview, which will help you keep track of the bounties you've paid out.
 
 For technical questions or help with your implementation, please reach out to support@hackerone.com or your HackerOne program manager.
+
+### Bounty Calculator
+
+When your program has a bounty table set up, you will be presented with the bounty calculator when awarding a bounty. The bounty calculator computes the amount according to the table set up for your program, based on the reports’ asset and severity.
+
+![bounty calculator](./images/bounties-1.png)
```
Comment (Reigertje) [inline docs/programs/bounties.md:86]: I believe this should use the bounties-2 image 🙂

**Review by Reigertje — CHANGES_REQUESTED** [review]

**Review by Reigertje — APPROVED** [review]

**Review by saida — APPROVED** [review]

**Review by 1Elle1111 — COMMENTED** [review]
Hi

**Review by 1Elle1111 — COMMENTED** [review]
Hi

**Pull request merged**